pkg/member/member_control.go

```diff
@@ -46,7 +46,7 @@ type Control interface {
 	PromoteMember(context.Context) error
 
 	// UpdateMemberPeerURL updates the peer address of a specified etcd cluster member.
-	UpdateMemberPeerURL(context.Context, client.ClusterCloser) (string, error)
+	UpdateMemberPeerURL(context.Context, client.ClusterCloser) error
```

Comment:
It is not clear on why this change has been made. If the UpdateMemberPeerURL returns an error then the status of TLS should be unknown which can also be explicitly set. Druid can then appropriately handle this state and not take any action and instead re-queue the item till a time it can determine the state.

Reply:
same reason https://github.com/gardener/etcd-backup-restore/pull/564#discussion_r1059844968